inc/html.class.php

```diff
@@ -2301,16 +2301,19 @@ static function getCheckbox(array $options) {
          }
       }
 
+      $label_class = "label-checkbox";
+
       if ($params['readonly']) {
          $out .= " disabled='disabled'";
+         $label_class .= " cursor-not-allowed";
```

Comment:
instead adding a class, you should rely in your css on the disabled attribute

Reply:
Done in 6c3d48ca6a573b51f88fb79b1910dec8ce551711.